src/app/static/src/js/utils/reportsTable.js

```diff
@@ -0,0 +1,84 @@
+function buildVersionBadge(data, full) {
+    if (data === full["latest_version"]) {
+        return '<span class="badge-info badge float-right">latest</span>'
+    }
+    else {
+        return '<span class="badge-light badge float-right">out-dated</span>'
+    }
+}
+
+function buildIdCell(data, type, full) {
+    if (!data || full["tt_parent"] === 0) {
```

Comment:
This code's getting repeated 3 times so i guess it could be pulled out as an isParentRow function. But also - why the need for the ||? When would the data be falsey for version fields if the row isn't a parent row?

Reply:
it shouldn't, you're right, I guess it's unnecessary